test/hotspot/jtreg/serviceability/jvmti/GetLocalVariable/GetSetLocalUnsuspended.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/**
+ * @test
+ * @summary Verifies JVMTI GetLocalXXX/SetLocalXXX return errors for unsuspended vthreads
+ * @requires vm.continuations
+ * @enablePreview
+ * @library /test/lib
+ * @run main/othervm/native -agentlib:GetSetLocalUnsuspended GetSetLocalUnsuspended
+ */
+
+
+public class GetSetLocalUnsuspended {
+    private static final String agentLib = "GetSetLocalUnsuspended";
+
+    static native void testUnsuspendedThread(Thread thread);
+
+    static private volatile boolean doStop;
+
+    static private void sleep(long millis) {
```

Comment:
Let's use default modifiers order
```suggestion
    private static void sleep(long millis) {
```

Reply:
Okay, fixed.
Thank you for the comment!